templates/base.html

```diff
@@ -0,0 +1,80 @@
+                <a class="site-title" rel="author" href="/">{{config.title}}</a>
+                <nav class="site-nav">
+                    <input type="checkbox" id="nav-trigger" class="nav-trigger">
+                    <label for="nav-trigger">
+                        <span class="menu-icon">
+                            <svg viewBox="0 0 18 15" width="18px" height="15px">
+                                <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"></path>
+                            </svg>
+                        </span>
+                    </label>
+                    <div class="trigger">{% block navlinks %}{% endblock navlinks %}</div>
+                </nav>
+                {% endblock header %}
+            </div>
+        </header>
+        <main class="page-content" aria-label="Content">
+            <div class="wrapper">
+                {% block content %}{% endblock content %}
+            </div>
+        </main>
+        <footer class="site-footer h-card">
+            <div class="wrapper">
+                {% block footer %}
+                <h2 class="footer-heading">{{config.title}}</h2>
+                <div class="footer-col-wrapper">
+                    <div class="footer-col footer-col-1">
+                        <ul class="contact-list">
+                            <li class="p-name">{{config.title}}</li>
+                            <li><a class="u-email" href="mailto:raph@levien.com">raph@levien.com</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="footer-col footer-col-2">
+                        <ul class="social-media-list">
+                            <li><a href="https://github.com/linebender"><svg class="svg-icon"><use xlink:href="/minima-social-icons.svg#github"></use></svg> <span class="username">linebender</span></a></li>
+                        </ul>
+                    </div>
+
+                    <div class="footer-col footer-col-3">
+                        <p>&copy; 2023 The Linebender Authors.</p>
+                        <p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This website's content is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.</p>
+                        <p>All code used in this website is available under the Apache-2.0 license.</p>
```

Comment:
This feels *slightly* off to me; in particular, what counts as code? Like, is it content in code blocks - what happens if someone moves some content between code blocks and not in a code block?

I think the simplest solution might be to just dual-license everything as both/either CC-BY and Apache, unless anyone else has a better suggestion. Whatever the final scheme is, we should probably describe it in a README  of this repo.

Reply:
I'll add an issue for this so we can resolve it in a subsequent PR.